WAMPClient.js

```diff
@@ -13,6 +12,7 @@ class WAMPClient {
 	}
 
 	static generateSignature(procedureCalls) {
+		/* eslint no-bitwise: ["error", { "int32Hint": true }] */
```

Comment:
Prefer adding this as a rule or even submitting a PR to https://github.com/LiskHQ/eslint-config-lisk-base
Maybe we just want to allow bitwise operators in general?